midi_rag.py

```
@@ -0,0 +1,333 @@
+# midi_rag.py - MIDI RAG 분석기
+
+import sys
+import os
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)